portfolio.js

```diff
@@ -0,0 +1,141 @@
+const portfolio = [
+  {
+    gridClass: 'grid1',
+    heading: 'My Recent Works',
+    h1Class: 'crete w400 s40 lh52 blackish',
+  },
+
+  {
+    gridClass: 'grid2',
+    imgLink: 'img/projectImg1.jpg',
+    heading: 'Multi-Post Stories',
+    h1Class: 'crete w400 s32 lh44 nBlue1',
+    textContent: "A daily selection of privately personlized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever sincr the 1500s, when an unknown printer took a standard dummy text",
+    textClass: 'inter w400 s16 lh24 nBue1',
+    ulClass: 'techList',
+    tags: ['css', 'html', 'bootstrap', 'Ruby'],
+    tagClass: 'project1Tag',
+    buttonText: 'See Project',
+  },
+
+  {
+    divClass: 'project2',
+    gridClass: ['grid3', 'grid4', 'grid5', 'grid6', 'grid7', 'grid8'],
+    heading: 'Professional Art <br> Printing Data',
+    h1Class: 'crete w400 s32 lh44 white p16',
+    textContent: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    textClass: 'inter w400 s16 lh24 white p16',
+    ulClass: 'techlist p16',
+    tags: ['css', 'html', 'bootstrap', 'Ruby'],
+    tagClass: 'inter w500 s16 lh20 white project2Tag',
+    buttonText: 'See Project',
+  },
```

Comment:
- [ ] Great effort working on the pop-up modal of your project 👏 👏 . But I would suggest we stick to the project requirement as much as possible, From the project in your `Most recent work`, you have 7 cards but currently, in the `desktop version` you have only created two cards for pop-up functionality the remaining cards were not functioning well. Please kindly fix this by making all the cards have a pop-up modal functionality. 👍 👍 
> N.B: The `mobile version` of the pop-up modal for all the cards works just fine.

Reply:
Good morning. Please on the Figma design file, there aren't any 'see project' buttons, so where exactly should I attach the event listeners?